src/core/ext/transport/inproc/inproc_transport.cc

```diff
@@ -166,17 +166,18 @@ struct inproc_stream {
       // Now transfer from the other side's write_buffer if any to the to_read
       // buffer
       if (cs->write_buffer_initial_md_filled) {
-        fill_in_metadata(this, &cs->write_buffer_initial_md,
-                         cs->write_buffer_initial_md_flags, &to_read_initial_md,
-                         &to_read_initial_md_flags, &to_read_initial_md_filled);
+        (void)fill_in_metadata(this, &cs->write_buffer_initial_md,
```

Comment:
I think Craig's work in #27714 will change this function to return void, so the changes in this file won't be needed anymore.

Reply:
Ack.